src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -356,14 +356,18 @@ const getClickConversionPayloadAndEndpoint = (
   // userIdentifierSource
   // if userIdentifierSource doesn't exist in properties
   // then it is taken from the webapp config
+  const { gbraid, wbraid, gclid } = payload.conversions[0];
   if (!properties.userIdentifierSource && UserIdentifierSource !== 'none') {
     set(payload, 'conversions[0].userIdentifiers[0].userIdentifierSource', UserIdentifierSource);
-
-    // one of email or phone must be provided
-    if (!email && !phone) {
+    // one of email or phone must be provided when none of gclid, wbraid and gbraid provided
+    if (!email && !phone && !(gclid || wbraid || gbraid)) {
```

Comment:
condition is updated but not the error message, ok?

Reply:
addressed.